lib/common/socket/uv-binding.c.h

```diff
@@ -175,22 +241,43 @@ h2o_socket_t *do_import(h2o_loop_t *loop, h2o_socket_export_t *info)
 
     if (sock == NULL)
         return NULL;
-    if (uv_tcp_open((uv_tcp_t *)sock->uv.stream, info->fd) != 0) {
+    if (uv_tcp_open((uv_tcp_t *)sock->handle, info->fd) != 0) {
         h2o_socket_close(&sock->super);
         return NULL;
     }
 
     return &sock->super;
 }
 
+h2o_socket_t *h2o_uv_poll_create(h2o_loop_t *loop, int fd, uv_close_cb close_cb)
```

Comment:
Please rename the function to `h2o_uv__poll_create` considering the fact that the socket create by this function cannot be used with certain API, e.g. `h2o_socket_write`, `h2o_socket_getpeername`, etc.

Note: we generally use `__` to designate a private function.